src/platform/file_manager.c

```diff
@@ -498,3 +499,12 @@ int platform_file_manager_close_file(FILE *stream)
 #endif
     return result;
 }
+
+bool platform_file_manager_has_stat(void)
+{
+#ifdef USE_FILE_CACHE
+    return platform_file_manager_cache_has_stat();
+#else
+    return true;
```

Comment:
No `true`/`false` please. Use just `0` and `1`.

Reply:
```suggestion
    return 1;
```